cfpb/cfgov-refresh
# Spacing adjustments to mobile secondary nav & form labels/text inputs

In accordance with DM/cfgov specs, adjusts spacing between heading labels and their text inputs, adds helper text formatting in email signup, and decreases the bottom margin of the secondary nav on mobile.


## Changes

- Reduce bottom spacing on secondary nav on mobile
- Reduce spacing between labels with H4 styling (`.a-label__heading`) and their text inputs. This affects several forms across the site: email signups, filterable list controls, conference registration, etc
- Remove `required` messaging in email signup (only necessary for `optional` fields)
- Reduce bottom margin in `m-form-field-with-button`

## Testing

1. Check out branch
2. Navigate to a page with [secondary nav](https://www.consumerfinance.gov/policy-compliance/guidance/implementation-guidance/) and review the spacing between the nav and page content on mobile
3. Navigate to page with an email signup and review its spacing/styling

## Screenshots
### Email signup
#### Current

<img width="410" alt="screen shot 2018-04-12 at 7 49 11 am" src="https://user-images.githubusercontent.com/778171/38689437-8c835c36-3e30-11e8-92d5-93214106ff23.png">

#### Updated
<img width="416" alt="screen shot 2018-04-26 at 8 40 38 am" src="https://user-images.githubusercontent.com/778171/39318110-18efec28-4932-11e8-92e8-5401c933ddcb.png">

### Blog filter
#### Current
<img width="810" alt="screen shot 2018-04-16 at 9 30 29 am" src="https://user-images.githubusercontent.com/778171/38837350-738345ee-4186-11e8-9cb3-bc298fcf0846.png">


#### Updated 
Spacing between the `Item name` and `From`/`To` labels & their text inputs is reduced
<img width="792" alt="screen shot 2018-04-16 at 2 59 04 pm" src="https://user-images.githubusercontent.com/778171/38837402-b4775f40-4186-11e8-827f-a154dec733b0.png">


### Secondary nav on mobile
#### Current
<img width="385" alt="screen shot 2018-04-12 at 9 05 25 am" src="https://user-images.githubusercontent.com/778171/38689500-b2c53eb4-3e30-11e8-826d-834e7c423ddf.png">

#### Updated
<img width="387" alt="screen shot 2018-04-12 at 7 52 28 am" src="https://user-images.githubusercontent.com/778171/38689512-b8437db0-3e30-11e8-9abb-2b531da8bf9d.png">

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
